OracleGoldenGate/setupLinuxEnv.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1982-2016 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: December, 2016
+# Author: gerald.venzl@oracle.com
```

Comment:
The header of this file needs changing as it has nothing to do anymore with database installation. And I also don't want to get emails about GoldenGate setups. :)